docs/getting_started/azure.md

```diff
@@ -0,0 +1,228 @@
+
+```bash
+$ az storage account keys list --account-name kopstest
+[
+  {
+    "keyName": "key1",
+    "permissions": "Full",
+    "value": "RHWWn..."
+  },
+  {
+    "keyName": "key2",
+    "permissions": "Full",
+    "value": "..."
+  }
+
+]
+
+$ export AZURE_STORAGE_KEY="RHWWn...“
+```
+
+
+Then create a blob container.
+
+```bash
+$ az storage container create --name cluster-configs
+{
+  "created": true
+}
+```
+
+You can confirm that the container has been successfully created from
+Storage Exporter or via `az storage container list`.
+
+```bash
+$ az storage container list --output table
+Name             Lease Status    Last Modified
+---------------  --------------  -------------------------
+cluster-configs  unlocked        2020-10-06T21:12:36+00:00
+```
+
+Set the env var `KOPS_STATE_STORE` to the container name URL using kOps' `azureblob://` protocol.
+The URL may include a path within the container.
+kOps stores all of its cluster configuration within this path.
+
+```bash
+export KOPS_STATE_STORE=azureblob://cluster-configs
+## Step 3. Set up Credentials for kOps
+
+Use the following commands to generate kOps credentials.
+
+First, create a service principal in Active Directory.
+
+```bash
+$ az ad sp create-for-rbac --name kops-test --role owner --sdk-auth
+
+{
+  "clientId": "8c6fddb5...",
+  "clientSecret": "dUFzX1...",
+  "subscriptionId": "7e232...",
+  "tenantId": "76253...",
+  ...
+}
+```
+
+Set corresponding env vars:
+
+- Set `AZURE_TENANT_ID` to the `tenantId` of the output
+- Set `AZURE_CLIENT_ID` to the `clienteId` of the output
+- Set `AZURE_CLIENT_SECRET` to the `clientSecret` of the output.
+
+```bash
+$ export AZURE_TENANT_ID="76253..."
+$ export AZURE_CLIENT_ID="8c6fddb5..."
+$ export AZURE_CLIENT_SECRET="dUFzX1..."
+```
+
+## Step 4. Run kOps Commands
+
+Use the following command to create cluster configs in the blob container.
+The command line flags prefixed with `--azure-` are for
+Azure-specific configurations.
+
+```bash
+$ export KOPS_FEATURE_FLAGS=AlphaAllowAzure
+
+$ kops create cluster \
+  --cloud azure \
+  --name my-azure.k8s.local \
+  --zones eastus-1 \
+  --network-cidr 172.16.0.0/16 \
+  --networking calico \
+  --azure-subscription-id "${AZURE_SUBSCRIPTION_ID}" \
+  --azure-tenant-id "${AZURE_TENANT_ID}" \
+  --azure-resource-group-name kops-test \
+  --azure-route-table-name kops-test \
+  --azure-admin-user ubuntu
+```
+
+Confirm that config files are created in Blob storage.
+
+```bash
+$ az storage blob list --container-name cluster-configs --output table
+```
+
+Use the following command to preview the Azure resources
+kOps will create for the k8s cluster.
+
+\```
+$ kops update cluster  \
+  --name my-azure.k8s.local
+\```
+
+Now add the `--yes` flag to have kOps provision the resources
+and create the cluster. This will also add a kubeconfig context
+for the cluster.
+
+\```
+$ kops update cluster  \
+  --name my-azure.k8s.local \
+  --yes
+\```
```

Comment:
these code blocks arent working because of the `\` in front of the triple backticks

Reply:
fixed!